### PR TITLE
Initial Commit for third party notice file

### PR DESCRIPTION
**Pull request by PureWeen**
### Description of Change

Add Third Party Notice file for all 3rd party libraries that we use. This adds the TPN txt file into every NUPKG file that we generate. 

These are the libraries that I found. 
  - Bootstrap
  - Mono
  - Open Sans fonts
  - glide
  - Open Iconic
  - gradle

### CODE REVIEW

File: Directory.Build.props [hunk 1]
```diff
@@ -143,6 +143,8 @@
     <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
     <PackageVersion>$(DotNetVersionBand)-dev</PackageVersion>
     <PackageOutputPath>$(MSBuildThisFileDirectory)artifacts</PackageOutputPath>
+    <LicenseFile>$(MSBuildThisFileDirectory)LICENSE.TXT</LicenseFile>
```
Comment (PureWeen) [inline Directory.Build.props:146]: Does this include it too many places? Graphics, Blazor, etc...

File: Directory.Build.props [hunk 2]
```diff
@@ -143,6 +143,8 @@
...
+    <PackageThirdPartyNoticesFile>$(MSBuildThisFileDirectory)THIRD-PARTY-NOTICES.TXT</PackageThirdPartyNoticesFile>
```
Comment (PureWeen) [inline Directory.Build.props:147]: Does this include License and TPN too many places? Graphics, Blazor, Blazor (WPF), Blazor (FORMS), etc...

Comment (Eilon) [reply Directory.Build.props:147]: I'd rather over include than under include. This is fine as far as I'm concerned. If we make it more complicated and try to slice and dice things we'll probably end up missing too many places either now or eventually.

**Review by Eilon — APPROVED** [review]
Assuming you've verified the NUPKG file has the actual TXT files in it, ship it!

**Comment by PureWeen** [comment]
> Assuming you've verified the NUPKG file has the actual TXT files in it, ship it!

I checked a few of them from CI 

image

image

image

**Comment by Eilon** [comment]
@PureWeen yeah that's perfect!

**Review by rmarinho — APPROVED** [review]

**Comment by rmarinho** [comment]
Do we need to backport this to net7 ?

**Comment by Eilon** [comment]
> Do we need to backport this to net7 ?

Seems reasonable and prudent.

**Pull request merged**